.github/autosubmit.yml

```diff
@@ -0,0 +1,16 @@
+default_branch: 'main'
+issues_repository:
+  owner: 'flutter'
+  repo: 'flutter'
+auto_approval_accounts:
```

Comment:
Do we need to split this config into two different places? depending on how this file will be read there are several dangerous values:

Assuming the file will be read from the PR commit:

* I can change the auto_approval_accounts to include my username and bypass approval
* I can change the # of approvals to 0 to bypass codereview
* I can change run_ci or required_checkruns to land immediately

Assuming the file will be read from main ToT:

* This will require adding owners approval to the autosubmit.yml to ensure someone who understands the implications of the values reviews changes to it, although it can still be error prone.

Maybe we can read the file from .github repository instead. This repo is usually well protected because it contains org level configurations and only allow to override non dangerous values per repository.

Reply:
Yeah I thought of this as well. Right now I just want to get it running as there are quite a few changes that needed to come in order to make this work. But as far as your options I am definitely not reading from the PR commit. There are several reasons why this would not be beneficial for us. I chose to always read from ToT to reduce network calls and to utilize caching. I am fine reading from .github repo as that is the same amount of network calls thought it does prevent other teams from adopting more quickly and would require them to have access to that repo correct?

Reply:
Not necessarily, usually .github is readonly for org members and write access to a few individuals. Everybody can send PRs but only the people with write access can +2 and submit.

Let's give it a try, can you try to create a PR in https://github.com/flutter/.github?

Reply:
Yeah let me try that.